test/data/dockerfiles/TestLabels

```diff
@@ -6,7 +6,7 @@
 
 # Pull base image.
 FROM dockerfile/ubuntu
-MAINTAINER ubuntu
+
```

Comment:
Are you OK with this change? Since my change is going to throw an `info` output rather than 'check passed'. Additionally, the MAINTAINER instruction is not used here in the nginx Dockerfile.